package.json

```diff
@@ -36,9 +37,14 @@
     "react-dom": "^18.2.0",
     "react-icons": "^4.8.0",
     "react-loader-spinner": "^5.3.4",
+    "react-redux": "^8.0.5",
     "react-router-dom": "^6.9.0",
     "react-scripts": "5.0.1",
     "react-toastify": "^9.1.1",
+    "redux": "^4.2.1",
+    "redux-persist": "^6.0.0",
+    "redux-persist-transform-filter": "^0.0.22",
+    "reselect": "^4.1.7",
```

Comment:
Just a note.
I'd recommend not to use 3rd party libraries for learning purposes.
You should at least once try native solutions like plain redux + react-redux.
Because it important to understand how they works without shortcuts.

Reply:
Using of redux-persist has been removed